src/components/SelectionList/index.tsx

```diff
@@ -32,7 +32,10 @@ function SelectionList<TItem extends ListItem>(props: BaseSelectionListProps<TIt
             // eslint-disable-next-line react/jsx-props-no-spreading
             {...props}
             ref={ref}
-            onScroll={() => {
+            onScroll={(event) => {
```

Comment:
destructure `onScroll` from props, because when we the spread: `{...props}` we also pass `onScroll`